inspire/dojson/schemas/hep-0.0.1.json

```diff
@@ -0,0 +1,646 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "properties": {
+        "abstracts": {
+            "items": {
+                "properties": {
+                    "source": {
+                        "enum": ["arXiv", "HEPDATA", "IOP", "APS", "Springer", "Elsevier", "AIP", "JACoW", "World Scientific", "WSP"]
```

Comment:
@tiborsimko, @egabancho, @jirikuncar: here is first question on best practice related to jsonschema adoption. How do we best integrate `KB`s in a reasonable way? In principle we could assume writing them directly into the given `jsonschema`. Alternatively we could refer to e.g.:
```json
{
    "$ref": "http://inspirehep.net/kb/source.json"
}
``` where the provided URL responds in JSONSchema. 
WDYT?